ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/StorageBackendTestsDBLock.scala

```diff
@@ -153,4 +154,20 @@ private[backend] trait StorageBackendTestsDBLock extends Matchers with StorageBa
       Future.successful(1 shouldBe 1)
     }
   }
+
+  def eventually[T](t: => T): T = {
```

Comment:
I’m a bit confused why you need this. Why does scalatest’s `eventually` not do the trick? That should work for both futures and non-future stuff.

Reply:
no confusion, I thought that is only for futures, thanks! changed it!